docs/datasets.md

```diff
@@ -267,22 +267,28 @@ The structure of the text detection dataset directory is organized as follows.
 
 - For `SynthText`:
   - Step1: Download `SynthText.zip` from [homepage](https://www.robots.ox.ac.uk/~vgg/data/scenetext/)
-  - Step2: Download [shuffle_labels.txt](https://download.openmmlab.com/mmocr/data/mixture/SynthText/shuffle_labels.txt)
-  - Step3: Download [instances_train.txt](https://download.openmmlab.com/mmocr/data/mixture/SynthText/instances_train.txt)
-  - Step4:
+  - Step2:
 
   ```bash
+  mkdir SynthText && cd SynthText
+  mv /path/to/SynthText.zip SynthText
   unzip SynthText.zip
-
-  cd SynthText
+  mv SynthText synthtext
```

Comment:
why not mkdir synthtext directly?

Reply:
To be compatible with the creepy shuffle_labels.txt :P

Reply:
to be compatible with the dataset structure presented above